cache_ui/cinfo.go

```diff
@@ -0,0 +1,122 @@
+package cache_ui
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"hash/crc32"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	CInfo struct {
+		Store            Store
+		cksum            uint32
+		buffSynced       []byte
+		cksumSyncedAstat uint32
+	}
+
+	Store struct {
```

Comment:
Same as above - do not export.
```suggestion
	store struct {
```